tsdb/chunkenc/byte_test.go

```diff
@@ -9,13 +9,19 @@ import (
 )
 
 func TestLoadBytesChunk(t *testing.T) {
-	// tree samples added (0,conprof) (1,conprof) (2,conprof)
+	//c := NewBytesChunk()
```

Comment:
forgot to remove?

Reply:
This is how the static `byte{}` was generated. I can add a comment if you want me to. This is just so that we can check if `LoadBytesChunk()` works correctly.